### DISCUSSION TITLE
API calls

### DISCUSSION THREAD

**Discussion by asit-prakash** [sushrut111/cowin-automation-extn#22] [General]
Since I have selected the auto-refresh option which means the extension will automatically fetch the latest search results in every 2 seconds, I have 2 questions regarding this.
1. I am not able to see the API calls for search in the browser, is it something that is running through the extension?
2.  According to the gov API rules they might block IPs if there are more than 100 requests coming within 5 minutes, doesn't it affect the user as we are hitting it every 2 seconds?

**Comment by sushrut111** [comment]
Hey @asit-prakash 
I apologise for delay in the response. 
1. Is this still observed?
2. Setting polling interval to 2 seconds would cause throttling. There in the subsequent releases, we provided an option to set a custom interval.

**Reply by asit-prakash** [reply]
@sushrut111 
No the problem I mentioned in point 1 doesn't exist anymore and I got the update to increase the refresh time after that throttling issue also resolved.

Thanks, man I was able to book a slot for my family using this.
